script/yank_user

```diff
@@ -14,11 +14,13 @@ require_relative "../config/environment"
 
 user = User.where(handle: handle).first
 
+security_user = User.where(email: "security@rubygems.org").first
```

Comment:
What about `find_by` for early fail (same can be used for user lookup above)?

Reply:
updated